src/workshop.md

```diff
@@ -1,46 +1,54 @@
 # MFEM Community Workshop
-#### October 26, 2023
-#### Virtual Meeting
-
-**Speakers' slides** are linked in the agenda below. Read the
-[**article**](https://computing.llnl.gov/about/newsroom/mfem-workshop-2023)
-about the workshop on LLNL's Computing website. Watch the
-[**video playlist**](https://www.youtube.com/playlist?list=PLy9rIbGDXrG0crap1EBFBRV_-RrIDa1an) of
-workshop presentations (linked individually below and available on the [videos page](videos.md)),
-and view **contest submissions** in the [gallery](gallery.md).
+#### October 22-24, 2024
+#### LLNL + Virtual
```

Comment:
Maybe "In-Person at LLNL + Virtual" or "In-Person at LLNL + Hybrid Virtual Option"

Reply:
This shows on the side as a menu item so I shortened it not to stand out

Reply:
Changed to "In-person at LLNL + Virtual" which still fits